aurora-answers.rb

```diff
@@ -30,28 +27,19 @@ def mirror(list)
 
 def switchPairs(list)
 
-  result = []
-
   i = 0
   how_many_times = list.length/2
   how_many_times.times do
-    first = list[i]
-    second = list[i+1]
-    result << second
-    result << first
+    list[i], list[i+1] = list[i+1], list[i]
```

Comment:
Ahh, fun! This is maybe kinda ruby magic (in the sense you can't do this in other languages), but I like it. :)